app/src/main/java/com/aritra/notify/ui/screens/notes/addEditScreen/AddEditScreen.kt

```diff
@@ -129,6 +130,18 @@ fun AddEditScreen(
     var photoUri by remember { mutableStateOf(emptyList<Uri?>()) }
 
     var characterCount by remember { mutableIntStateOf(title.length + description.length) }
+
+
+    val titleWordCount = remember { derivedStateOf { countWords(title) } }
+    val desWordCount = remember { derivedStateOf { countWords(description) } }
+    var totalWordCount by remember {
+        mutableIntStateOf(titleWordCount.value + desWordCount.value)
```

Comment:
The `totalWordCount` will be only the `desWrordCount` not the addition of `title` & `des`